frontend/src/pages/arrange_components/arrange_components.js

```diff
@@ -1,27 +1,16 @@
-import React from 'react'
-
-import "react-grid-layout/css/styles.css";
-import "../../../node_modules/react-resizable/css/styles.css";
-import { Responsive, WidthProvider } from 'react-grid-layout';
-import VisualComponentsLayout from "./visual_components_layout";
-import styled from "styled-components";
-import axios from "axios";
-import {confirmAlert} from "react-confirm-alert";
-import {FontAwesomeIcon} from "@fortawesome/react-fontawesome";
-import {faQuestion} from "@fortawesome/free-solid-svg-icons";
-import "../../pages.css"
-require('dotenv').config();
+import '../../../node_modules/react-resizable/css/styles.css';
+import '../../pages.css';
+import 'react-grid-layout/css/styles.css';
+
+import { faQuestion } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import axios from 'axios';
+import React from 'react';
+import { confirmAlert } from 'react-confirm-alert';
 
-const ResponsiveGridLayout = WidthProvider(Responsive);
+import VisualComponentsLayout from './visual_components_layout';
 
-const Main = styled.main`
```

Comment:
Is this totally unused across the project?

Reply:
Yes, I think so. 

There is just one "Main" component used and this one is definied in the component itself.
![image](https://user-images.githubusercontent.com/34272764/79725455-8a72ed80-82e9-11ea-8ae7-8bcfb8e401f3.png)